.gitignore

```diff
@@ -0,0 +1,21 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Since this is a Flask app, what about ignoring Flask-specific files?

Example:

```
instance/
.webassets-cache
```

Reply:
Added `instance/`, adding `.webassets-cache` won't be necessary as we don't use `webassets`/`Falsk-Assets`